### PR TITLE
Added the ability to teleport straight to warp

### PR DESCRIPTION
**Pull request by Lukatjee**
Attempted the requested changes: https://github.com/BG-Software-LLC/SuperiorSkyblock2/pull/410

### CODE REVIEW

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java [hunk 1]
```diff
@@ -58,13 +61,52 @@ public void execute(SuperiorSkyblockPlugin plugin, CommandSender sender, String[
         Pair<Island, SuperiorPlayer> arguments = args.length == 1 ? CommandArguments.getSenderIsland(plugin, sender) :
                 CommandArguments.getIsland(plugin, sender, args[1]);
 
+        System.out.println(args[0]);
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java]: This should be removed.

**Review by OmerBenGera — CHANGES_REQUESTED** [review]

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java (hunk 1 above)
Comment (Lukatjee) [reply src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java]: Removed that line

**Comment by JamesSmily** [comment]
Jesus died cause of the fucking scum bag jews that killed him and i'm glad hitler murdered those scum bags like omre and his family. Boy got mad cause i said jesus got killed by the jews which he is also jewish and banned me from the discord and that is why the jews got killed cause of being scum like him. No money poor af living in a poor country relying on minecraft for money but even that didn't work so he quit cause he's depressed.

**Comment by JamesSmily** [comment]
Jesus died cause of the fucking scum bag jews that killed him and i'm glad hitler murdered those scum bags like omre and his family. Boy got mad cause i said jesus got killed by the jews which he is also jewish and banned me from the discord and that is why the jews got killed cause of being scum like him. No money poor af living in a poor country relying on minecraft for money but even that didn't work so he quit cause he's depressed.

**Comment by Skizzles** [comment]
@OmerBenGera is this good to merge?

**Comment by OmerBenGera** [comment]
I still need to do tests, so no

**Review by OmerBenGera — CHANGES_REQUESTED** [review]
Amazing job, I tested it and it seems to work flawlessly fine.
It's only missing one thing tho - tab completes don't support the warp names yet.
I'd like it to also have tab completes support, so it will show only warp names that you can teleport to.

After that it is added, I will merge the changes.
Good work bud!

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java [hunk 2]
```diff
@@ -103,14 +103,21 @@ private void warpPlayer(IslandWarp warp, SuperiorPlayer superiorPlayer, Island i
...
 
     }
 
     @Override
     public List<String> tabComplete(SuperiorSkyblockPlugin plugin, CommandSender sender, String[] args) {
-        return args.length == 2 ? CommandTabCompletes.getPlayerIslandsExceptSender(plugin, sender, args[1],
-                plugin.getSettings().tabCompleteHideVanished) : new ArrayList<>();
+
+        return switch (args.length) {
+            case 2 -> CommandTabCompletes.getPlayerIslandsExceptSender(plugin, sender, args[1],
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java]: You can't use Java 16 features.

**Review by OmerBenGera — CHANGES_REQUESTED** [review]

**Comment by Lukatjee** [comment]
Any updates on this @OmerBenGera ?

**Comment by OmerBenGera** [comment]
> 
> 
> Any updates on this @OmerBenGera ?

I had to fix some issues with tab completes, but other than that everything was fine and I merged it :)

**Pull request merged**